src/adv.py

```diff
@@ -38,14 +44,39 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player(room['outside'], 'Bart', [])
 
-# Write a loop that:
-#
-# * Prints the current room name
-# * Prints the current description (the textwrap module might be useful here).
-# * Waits for user input and decides what to do.
-#
-# If the user enters a cardinal direction, attempt to move to the room there.
-# Print an error message if the movement isn't allowed.
-#
+
+# Prints the current room name, description, and items in the room.
+# Waits for user input and decides what to do
+# Check if command is 1 word.
 # If the user enters "q", quit the game.
+# If the user enters "i" or "inventory" show player's current inventory.
+# Check if command is 1 words: if it is, check if word is a valid direction, if not send error message.
+# If user enters valid direction: n, s, e, w then run handle_direction
+# Check if command is 2 words: if it is, check that the two word combination is valid, if not send message.
+# If two word combination is valid then run get_item and then try handle_action.
+
+while True:
+    print(player.current_room)
+    string = input('\nEnter a command: ').lower().split()
+    if len(string) == 1:
+        if string[0] == 'q':
+            print("\nWe'll see you next time, adventurer.")
+            break
+        elif string[0] == 'i' or string[0] == 'inventory':
+            inventory = [i.name for i in player.inventory_list]
+            print(f"\n Your current inventory: {', '.join(inventory)}\n")
+        else:
+            player.current_room = handle_direction(string[0], player.current_room)
```

Comment:
You need to have a case for if the user inputs bad data. So you can do something like tell them available commands or however you think you should inform them.